docs/registry.md

```diff
@@ -71,5 +72,63 @@ If the module is successfully registered, you can use this converter through con
 
 ```python
 converter_cfg = dict(type='Converter1', a=a_value, b=b_value)
-converter = build_converter(converter_cfg)
+converter = CONVERTERS.build(converter_cfg)
+```
+
+## Hierarchy Registry
```

Comment:
This part needs refactoring. Users will get confused by this doc and do not know why and how.